### PR TITLE
Update game

### PR DESCRIPTION
**Pull request by vaibhavsharma2611**
Update Game

### CODE REVIEW

**Review by vaibhavsharma2611 — COMMENTED** [review]
Update Game Game

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (vaibhavsharma2611) [inline index.html:50]: Test Suggestions

Comment (Vaibhav-2611) [inline index.html:50]: Test

**Review by Vaibhav-2611 — COMMENTED** [review]
Test

**Review by vaibhavsharma2611 — COMMENTED** [review]
Test

**Pull request merged**